diff --git a/node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m b/node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m
index c139440..13d5e59 100644
--- a/node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m
+++ b/node_modules/@react-native-community/datetimepicker/ios/RNDateTimePickerShadowView.m
@@ -5,7 +5,9 @@ @implementation RNDateTimePickerShadowView
 - (instancetype)init
 {
   if (self = [super init]) {
-    YGNodeSetMeasureFunc(self.yogaNode, RNDateTimePickerShadowViewMeasure);
+    //YGNodeSetMeasureFunc(self.yogaNode, RNDateTimePickerShadowViewMeasure);
+    YGNodeSetMeasureFunc(self.yogaNode, (YGMeasureFunc)RNDateTimePickerShadowViewMeasure);
+
   }
   return self;
 }
